packages/@react-aria/interactions/docs/interactions.mdx

```diff
@@ -112,5 +112,5 @@ mouse input, touchscreens, and also hybrid devices.
 * Firefox 76+ on macOS and Windows
 * Safari 12+ on macOS
 * Edge 44+ on Windows
```

Comment:
Let's make this match Chrome for now. We should probably bump the Chrome version at some point, but idk to what. Just want to convey that we don't support old non-Chromium Edge.

```suggestion
* Edge 80+ on Windows
```